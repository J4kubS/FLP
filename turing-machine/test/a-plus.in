S a S R
S a T R
T   F  
aaaaaaaaaa
