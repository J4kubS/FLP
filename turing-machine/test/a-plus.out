Saaaaaaaaaa
aSaaaaaaaaa
aaSaaaaaaaa
aaaSaaaaaaa
aaaaSaaaaaa
aaaaaSaaaaa
aaaaaaSaaaa
aaaaaaaSaaa
aaaaaaaaSaa
aaaaaaaaaSa
aaaaaaaaaaT
aaaaaaaaaaF 
